src/System.Net.Requests/src/Resources/Strings.resx

```diff
@@ -265,4 +265,7 @@
   <data name="net_ftpstatuscode_ActionNotTakenFilenameNotAllowed" xml:space="preserve">
     <value>File name not allowed</value>
   </data>
+  <data name="net_readonlystream" xml:space="preserve">
+    <value>The stream does not support writing.</value>
+  </data>
```

Comment:
Removing it. Initially I had written an exception to be thrown if the stream is not writable. But that didn't make sense as the code was knowingly using a writable MemoryStream, so the exception was redundant, but then forgot to scrap this after I removed the exception.